package.json

```diff
@@ -6,32 +6,30 @@
   "author": "Mikołaj Klaman <mklaman@virtuslab.com>",
   "license": "MIT",
   "private": false,
-  "main": "dist/main/index.js",
-  "module": "dist/module/index.js",
-  "esnext": "dist/esnext/index.js",
-  "typings": "dist/main/index.d.ts",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
   "engines": {
     "node": "12.18.x",
     "yarn": "1.22.x"
   },
```

Comment:
I am wondering if we should remove this section - I remember having some problem installing the package becasue of mismatching node version